Sources/Extensions/Foundation/UserDefaultsExtensions.swift

```diff
@@ -38,4 +38,33 @@ public extension UserDefaults {
 		return object(forKey: key) as? Date
 	}
 	
+    /// SwifterSwift: Retrieves a Codable object from UserDefaults.
+    ///
+    /// - Parameters:
+    ///   - type: Class that conforms to the Codable protocol.
+    ///   - key: key to find the Codable object for.
+    /// - Returns: Codable object for key (if exists).
+    public func object<T: Codable>(_ type: T.Type, with key: String) -> T? {
+        if let data = self.value(forKey: key) as? Data {
```

Comment:
We can just use the `try?` keyword to ignore the thrown error and return `nil` 😛 
```Swift
let data = self.value(forKey: key) as? Data else { return nil }
return try? JSONDecoder().decode(type.self, from: data)
```

Reply:
Yes yes! I agree. Will update 😄